src/Create.js

```diff
@@ -65,75 +69,77 @@ export default function Create({ onSubmit, upload }) {
-        <StyledSelect
-          type="text"
-          onChange={onInputChange}
-          value={data.category}
-          name="category"
-          data-cy="select-one"
-        >
-          <option>Wähle eine Option aus</option>
-          <option value="Gemüse">Gemüse</option>
-          <option value="Frucht">Obst</option>
-          <option value="Aufschnitt">Aufschnitt</option>
-          <option value="Trockenes">Trockenes</option>
-        </StyledSelect>
-        <StyledText>Was möchtest du retten?</StyledText>
-        <StyledInput
-          type="text"
-          name="title"
-          value={data.title}
-          onChange={onInputChange}
-          data-cy="input-one"
-        />
-        <StyledText>Wie riecht&apos;s?</StyledText>
-        <StyledSelect
-          type="text"
-          name="smell"
-          value={data.smell}
-          onChange={onInputChange}
-          data-cy="select-two"
-        >
-          <option>Wähle eine Option aus</option>
-          <option value="Einfach lecker!">Einfach lecker!</option>
-          <option value="Gut">Gut</option>
-          <option value="Okay">Okay</option>
-          <option value="Schlecht">Ab auf den Komposthaufen</option>
-        </StyledSelect>
-        <StyledText>Wie sieht&apos;s aus?</StyledText>
-        <StyledSelect
-          type="text"
-          name="optic"
-          value={data.optic}
-          onChange={onInputChange}
-          data-cy="select-three"
-        >
-          <option>Wähle eine Option aus</option>
-          <option value="Top">Top</option>
-          <option value="Noch ganz gut">Noch ganz gut</option>
-          <option value="Bio-Tonne">Bio-Tonne</option>
-        </StyledSelect>
-        <StyledText>Wo kann man es abholen?</StyledText>
-        <StyledInput
-          type="text"
-          name="location"
-          value={data.location}
-          onChange={onInputChange}
-          data-cy="input-two"
-        />
-        <StyledButton>Save me</StyledButton>
-      </form>
-      <StyledButton onClick={() => console.log(data)} />
-    </StyledSection>
+    <StyledForm onSubmit={onSubmitHandler}>
+      <div>
+        {data.image ? (
+          <img src={data.image} alt="" style={{ width: '100%' }} />
+        ) : (
+          <input type="file" name="file" onChange={fileHandler} />
+        )}
+      </div>
+      <StyledLabel for="select-category">Wähle eine Kategorie aus:</StyledLabel>
+      <StyledSelect
+        type="text"
+        onChange={onInputChange}
+        value={data.category}
+        name="category"
+        data-cy="select-one"
+        id="select-category"
+      >
+        <option>Wähle eine Option aus</option>
+        <option value="Gemüse">Gemüse</option>
+        <option value="Frucht">Obst</option>
+        <option value="Aufschnitt">Aufschnitt</option>
+        <option value="Trockenes">Trockenes</option>
+      </StyledSelect>
+      <StyledLabel for="input-title">Was möchtest du retten?</StyledLabel>
```

Comment:
here you need to do something with "for" since it's throwing errors around like a pimp bucks in the club